src/canvas.tsx

```diff
@@ -206,6 +269,7 @@ export const Canvas = React.memo(
 
         for (let hit of hits) {
           let stopped = { current: false }
+
           fn({
```

Comment:
Given my limited knowledge of `three.js` it was hard to type the Object that is passed back in the `handleIntersects` callback fn. I've left this alone for now, but would love to revisit this.